.github/workflows/release.yml

```diff
@@ -0,0 +1,57 @@
+name: Release
+
+# Runs manually
+on:
+  workflow_dispatch:
+      
+jobs:
+  release:
+    name: Release on Sonatype OSS
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Set up Apache Maven Central
+        uses: actions/setup-java@v3
+        with:
+          java-version: 11
+          distribution: 'temurin'
+          cache: 'maven'
+          server-id: ossrh # Value of the distributionManagement/repository/id field of the pom.xml
+          server-username: SONATYPE_USERNAME # env variable for username in deploy
+          server-password: SONATYPE_PASSWORD # env variable for password in deploy
+          gpg-private-key: ${{ secrets.GPG_PRIVATE_KEY }} # Value of the GPG private key to import
+          gpg-passphrase: GPG_PASSPHRASE # env variable for GPG private key passphrase
+
+      - name: Publish to Apache Maven Central
+        run: |
+          mvn -DskipTests deploy -Dgpg.passphrase=${{ secrets.GPG_PASSPHRASE }}
+        env:
+          SONATYPE_USERNAME: ${{ secrets.SONATYPE_USERNAME }}
+          SONATYPE_PASSWORD: ${{ secrets.SONATYPE_PASSWORD }}
+          GPG_PASSPHRASE: ${{ secrets.GPG_PASSPHRASE }}
+
+  # Create github release & tag
+  gitHub_release:
+    name: create github release
+    runs-on: ubuntu-latest
+    needs: release
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: create gitHub release
+        run: |
+          maven_config="$(cat /home/runner/work/devonfw-ide/devonfw-ide/.mvn/maven.config)"
```

Comment:
do we need to have this path absolute and know the location of `/home/runner/work/devonfw-ide/devonfw-ide/`?
There seems to be a lot of internal github action knowledge inside this.
`/home/runner` could at least be changed to `~` so it will not break if github decides to change the name of the linux user running the github action.
The `work` directory also seems to be internal knowledge.
And why is there `devonfw-ide` twice in the path?

What is PWD when you run the github action?
I would assume it is at least `/home/runner/work/devonfw-ide/`.
So we could also try to use a relative path to avoid coding all this magic internal github stuff...

Reply:
As I recall it didn't work when I was using the relative path and I didn't know why but now I tested it again and it works so no need for the absolute path.
devonfw-ide being there twice is how github actions does it. I already noticed that in other places.